packages/office-ui-fabric-react/src/components/Text/examples/Text.Weights.Example.tsx

```diff
@@ -0,0 +1,33 @@
+import { Text } from 'office-ui-fabric-react/lib/Text';
```

Comment:
Shouldn't this example show up in the PR deploy site? http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/12398/merge/fabric-website/dist/index.html?devhost#/controls/web/text

Reply:
Might need to add it to Text.doc.tsx